internal/bzlmod/default_gazelle_overrides.bzl

```diff
@@ -47,6 +50,9 @@ DEFAULT_DIRECTIVES_BY_PATH = {
     "github.com/googleapis/gnostic": [
         "gazelle:proto disable",
     ],
+    "github.com/pseudomuto/protoc-gen-doc": [
+        "gazelle:resolve go github.com/mwitkow/go-proto-validators @com_github_mwitkow_go_proto_validators//:validators_gogo",
```

Comment:
The reason for this override is github.com/mwitkow/go-proto-validators comes with its build files and the naming convention is not recognized by Gazelle. It will be as stable as the target names in github.com/mwitkow/go-proto-validators.